examples/vm_os_upgrade/06_setup_cloned_vm.yml

```diff
@@ -0,0 +1,16 @@
+- name: Report uptime
+  ansible.builtin.command: uptime
+
+- name: Upload index.html
+  ansible.builtin.copy:
+    src: index.html
+    dest: /var/www/html/index.html
```

Comment:
No. This must NOT be needed. The cloned VM must already contain the "application data".